base64ct/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 #![doc(
     html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
     html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
-    html_root_url = "https://docs.rs/base64ct/1.4.1"
```

Comment:
@tarcieri 
Considering that you forgot to bump it, I think it's worth to remove uses of `html_root_url` from all of our crates. :)

Reply:
Yeah I’ve been slowly doing that as I cut new releases. Guess I didn’t do this one yet.